Tests/Microsoft.AppCenter.Crashes.Test.Windows/ErrorAttachmentLogTest.cs

```diff
@@ -1,6 +1,7 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+using System;
```

Comment:
We've talked about this one offline. We should use the default format configuration that won't bring `System` usings to the top.

Reply:
not a blocker though